docker/ezbake/docker-entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -e
+
+if [ -n "$EZBAKE_REPO" ]; then
+  git clone $EZBAKE_REPO /ezbake
+  cd /ezbake
+  if [ -n "$EZBAKE_BRANCH" ]; then
+    git checkout origin/$EZBAKE_BRANCH
```

Comment:
Why does `origin` need to be specified? I'm wondering if the BRANCH/REF vars can be consolidated.

Reply:
I was having issues getting branches to actually be checked out without specifying origin, but I'll test again.

Reply:
yep, can confirm that when i pass a branch name to REF it doesn't actually check out a different branch/commit, just whatever the default is

Reply:
weird!